src/leap/mx/tcp_map.py

```diff
@@ -27,11 +30,14 @@
 TCP_MAP_CODE_PERMANENT_FAILURE = 500
 
 
-class LEAPPostfixTCPMapServerFactory(ServerFactory):
+class LEAPPostfixTCPMapServerFactory(ServerFactory, object):
```

Comment:
Out of curiosity, why import `object` here? I never got to understand what is the difference between importing object and not.